server/repository/src/db_diesel/encounter_row.rs

```diff
@@ -49,6 +50,7 @@ pub struct EncounterRow {
     /// The encounter document name
     pub document_name: String,
     pub program_id: String,
+    #[column_name = "patient_link_id"]
```

Comment:
Chris might have already mentioned. We need use the proper patient_link_id at some point otherwise we  might use a patient_link_id instead of patient_id which would cause the queries to fail...

So this would require to put the patient NameRow into the `pub type Encounter = (EncounterRow, ProgramRow);` type (or make it a proper struct). and then use the name row id instead of the patient_link_id from the EncounterRow...

Reply:
[5079280](https://github.com/msupply-foundation/open-msupply/pull/2744/commits/50792809ecce9ffaa1023a5cdf6c395e3c64bd57)